btk/draw_blends.py

```diff
@@ -470,29 +459,25 @@ def render_single(self, catalog_line, cutout, band):
         """
         k = int(np.random.rand(1) * len(self.cat))  # catalog_line["btk_index"][0]
         gal = self.cat.makeGalaxy(k, gal_type="real", noise_pad_size=0).withFlux(1)
```

Comment:
Does this work ? You removed the `self.cat` definition in the init, so I would expect the code to break here.

Reply:
Yeah I removed it because I thought we should be able to do this from the Catalog class instead of having to pass in another `galsim_catalog` to the DrawBlendsGenerator. I'll try to see how to make this work exactly soon

Reply:
I agree.